test/fixtures/asar-prebuilt/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "PrebuiltAsar",
+  "version": "0.0.1",
+  "description": "A test of the renamed from `electron-prebuilt` to `electron`",
+  "main": "main.js",
+  "devDependencies": {
+    "electron": "^1.8.0",
+    "electron-prebuilt": "^1.4.0"
```

Comment:
There should only be one item in `devDependencies`, and it should be `"electron": "1.3.1"`. Every time we add a new Electron version to the tests, it takes longer to run the tests due to downloading prebuilt binaries.